### DISCUSSION TITLE
Is it possible to release a new version with revive 1.2.3?

### DISCUSSION THREAD

**Discussion by ghost** [golangci/golangci-lint#3083] [Q&A] [answered]
https://github.com/mgechev/revive/issues/713 has been fixed in **1.2.2** and https://github.com/mgechev/revive/pull/727 in **1.2.3**.

This issue normally really fix https://github.com/golangci/golangci-lint/issues/2997 without to run **golangci-lint** in degraded mode (plugins disabled).

Thank you

**Comment by atc0005** [comment]
fwiw, I ran into a situation where I'm now seeing this error with golangci-lint v1.49.0 (which contains revive v1.2.3):

> `cmd/check_whois/main.go:8:1: package-comments: should have a package comment (revive)`

There is a doc.go file at the root of the project which provides package comments for package `main`.

I've not filed a bug report yet as I was planning to gather additional details and open one upstream first.

**EDIT**: Submitted as https://github.com/mgechev/revive/issues/741

**Comment by ldez** [comment, marked as answer]
v1.2.3 of revive has shipped in v1.49.0

https://github.com/golangci/golangci-lint/releases/tag/v1.49.0

**Comment by ghost** [comment]
Thank you very much ! :heart:
